packages/components/Tabs/src/Tabs.types.ts

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import { ITextProps } from '@fluentui-react-native/text';
+import type { IViewWin32Props } from '@office-iss/react-native-win32';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { IForegroundColorTokens, FontTokens, IBackgroundColorTokens } from '@fluentui-react-native/tokens';
+import { FocusZoneProps } from '@fluentui-react-native/focus-zone';
+import { View, ViewProps } from 'react-native';
+
+export const tabsName = 'Tabs';
+
+export interface ITabsContext {
+  /*
+   ** The currently selected TabsItem's key
+   */
+  selectedKey: string | null;
+
+  /*
+   ** Index of currently selected key
+   */
+
+  getTabId?: (key: string, index: number) => void;
+
```

Comment:
I thought this function was supposed to allow developers to customize how IDs are generated for each tab item (see [Fluent UI Pivot](https://developer.microsoft.com/en-us/fluentui#/controls/web/pivot)). If that's still the case, I'd probably add more details in the comment above it and make the return type 'string' instead of 'void'.

Reply:
Oh sorry I see that there's a description for it below under `TabsProps`. In that case, could you make the function return type 'string' instead of 'void'?